reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md

```diff
@@ -197,18 +222,20 @@ Accept wildcard characters: True
 ```
 
 ### -CommandType
-Exports only the specified types of command objects.
-Use "CommandType" or its alias, "Type".
 
-Valid values are:
+Exports only the specified types of command objects. Use **CommandType** or its alias, **Type**.
 
-- Alias: All Windows PowerShell aliases in the current session.
-- All: All command types. It is the equivalent of "get-command *".
-- Application: All files other than Windows PowerShell files in paths listed in the Path environment variable ($env:path), including .txt, .exe, and .dll files.
-- Cmdlet: The cmdlets in the current session. "Cmdlet" is the default.
-- ExternalScript: All .ps1 files in the paths listed in the Path environment variable ($env:path).
-- Filter and Function: All Windows PowerShell functions.
-- Script: Script blocks in the current session.
+The acceptable values for this parameter are as follows:
+
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```

Comment:
The accepted values list is correct but Configuration and Workflow are missing from the list above.

Reply:
Added the Accepted values to YAML and added Configuration and Workflow to the list with with links to about_Session_Configurations and about_Workflows.